app/controllers/reviews_controller.rb

```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+
+class ReviewsController < ApplicationController
+  respond_to :js
+  def create
+    @review = Review.new(review_params)
+    return flash.now[:success] = t('review.thanks_message') if @review.save
+    flash.now[:danger] = t('review.smth_went_wrong')
+  end
```

Comment:
Is not secured method
you can just you load resource and check ability from cancancan by 'load_and_authorize_resource'

Reply:
done